src/main/java/com/direwolf20/buildinggadgets/common/config/InGameConfig.java

```diff
@@ -0,0 +1,243 @@
+package com.direwolf20.buildinggadgets.common.config;
+
+import com.direwolf20.buildinggadgets.common.BuildingGadgets;
+import com.direwolf20.buildinggadgets.common.network.PacketHandler;
+import com.direwolf20.buildinggadgets.common.network.PacketSyncConfig;
+import com.direwolf20.buildinggadgets.common.tools.ReflectionTool;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.nbt.NBTBase;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.gameevent.PlayerEvent.PlayerLoggedInEvent;
+
+import java.lang.reflect.Field;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Mod.EventBusSubscriber(modid = BuildingGadgets.MODID)
+public class InGameConfig {
+    private static final String KEY_NAME = "NAME";
+    private static final String KEY_VALUE = "VAL";
+    /*
+    @SyncedConfig
```

Comment:
All of this could be removed?

Reply:
The idea is to sync it from the Server...
If it would be synced directly into the original config, the master copy would be lost

Reply:
I was unsure whether I should leave the test code in or not and hope that someone would complain, if it should be removed